### ISSUE TITLE
Air entry pressure (PTF) from SoilGrids

### ISSUE THREAD

**Issue by hboisgon** [Deltares/hydromt_wflow#28]
For the air entry pressure a fixed value is used in the wflow_sbm code, and we do not (yet) provide a map for this parameter. A PTF is available (Rawls and Brakensiek) for the Brooks-Corey air entry pressure value that we can add to setup_soilmaps.

**Comment by hboisgon** [comment]
@verseve is that issue still relevant for wflow Julia?

**Comment by verseve** [comment]
Yes, this is also still relevant for Wflow.jl, now we use a default value.
